### PR TITLE
Sc 12459/create tenant

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Adds SetupTenant view to the Beacon UI. 

Fixes SC-12459

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria
Screenshot 2023-01-27 at 5 59 10 PM (2)
Screenshot 2023-01-27 at 5 59 14 PM (2)

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/components/ui/SetupTenant/SetupTenant.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { memo } from 'react';
+
+import CustomizeTenant from './CustomizeTenant';
+import TenantHeader from './TenantHeader';
+import TenantQuickStart from './TenantQuickStart';
+
+function SetupTenant() {
+  return (
+    <section className="bg-hexagon bg-contain">
```
Comment (daniellemaxwell) [inline web/beacon-app/src/components/ui/SetupTenant/SetupTenant.tsx:9]: The Figma design includes the hexagons in the background, but they don't appear in Storybook. If the bg image should be removed, let me know.

File: web/beacon-app/src/components/ui/SetupTenant/TenantQuickStart.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <div className="max-w-sm p-10 text-center">
+        <h3 className="text-xl font-bold">Quick Start Tenant</h3>
+        <img src={bolt} alt="" className="mx-auto mt-5" />
+        <p className="mt-6">
+          We&#39;ll set up a tenant for you based on the closest region with availability x (by your
+          IP address) and label it as your <span className="font-bold">Development</span>{' '}
+          environment.
+        </p>
+        <p className="mt-6">You can change settings later and upgrade at any time.</p>
+        <Button className="mt-32 h-14 w-44 rounded bg-danger-500 text-center text-xl font-bold text-white">Create</Button>
```
Comment (daniellemaxwell) [inline web/beacon-app/src/components/ui/SetupTenant/TenantQuickStart.tsx:17]: I created a class for the repeated button styles in the CSS file, but the styling didn't appear in Storybook so I added it back to the className here.

File: web/beacon-app/src/components/ui/SetupTenant/CustomizeTenant.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <img src={world} alt="" className="mx-auto mt-5" />
+        <p className="mt-6">
+          You can customize your tenant settings, including regions (single and multi-region) and
+          cloud providers.
+        </p>
+        <p className="mt-6">
+          Customizing your tenant requires a paid plan. Upgrade now to set up a tenant specific to
+          your development and modeling needs.
+        </p>
+        <Button className="mt-16 h-14 w-48 rounded bg-danger-500 text-center text-xl font-bold text-white">Coming Soon</Button>
```
Comment (daniellemaxwell) [inline web/beacon-app/src/components/ui/SetupTenant/CustomizeTenant.tsx:19]: I created a class for the repeated button styles in the CSS file, but the styling didn't appear in Storybook so I added it back to the className here.

**Review by masskoder — APPROVED** [review]
LGTM

**Pull request merged**